src/libraries/System.Net.Sockets/src/System/Net/Sockets/Socket.cs

```diff
@@ -1375,6 +1377,42 @@ public int SendTo(ReadOnlySpan<byte> buffer, SocketFlags socketFlags, EndPoint r
             return bytesTransferred;
         }
 
+        /// <summary>
+        /// Sends data to a specific endpoint using the specified <see cref="SocketFlags"/>.
+        /// </summary>
+        /// <param name="buffer">A span of bytes that contains the data to be sent.</param>
+        /// <param name="socketFlags">A bitwise combination of the <see cref="SocketFlags"/> values.</param>
+        /// <param name="remoteSA">The <see cref="EndPoint"/> that represents the destination for the data.</param>
+        /// <returns>The number of bytes sent.</returns>
+        /// <exception cref="ArgumentNullException"><c>remoteEP</c> is <see langword="null" />.</exception>
+        /// <exception cref="SocketException">An error occurred when attempting to access the socket.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="Socket"/> has been closed.</exception>
+        public int SendTo(ReadOnlySpan<byte> buffer, SocketFlags socketFlags, SocketAddress remoteSA)
```

Comment:
Is "remoteSA" the agreed upon name for the parameter?

Reply:
no. The API proposal had `socketAddress`.  There was suggestion to name it `recievedSocketAddress` to avoid confusion with idea that the method only receives data from given address.  But that did not make it to final revision. 
I generally pick the name for consistency with original `remoteEP`.